nexus3/post_install.md

```diff
@@ -0,0 +1,78 @@
+## Sonatype Nexus Repository oss
+
+* Single source of truth for all of your components, binaries, and build artifacts.
+* Efficiently distribute parts and containers to developers.
+* Deployed at more than 100,000 organizations globally.
+
+### Get started
+
+[Sonatype Nexus Repository OSS](https://www.sonatype.com/products/repository-oss)
+
+[Sonatype Nexus Repository Documentation.](https://help.sonatype.com/docs?topnav=true)
+
+[Using Nexus Repository](https://blog.sonatype.com/using-nexus-3-as-your-repository-part-1-maven-artifacts)
+
+[Using Mirrors for Repositories with Maven](https://maven.apache.org/guides/mini/guide-mirror-settings.html)
+
+### Admin Credentials
+
+The nexus url is set with default admin credentials with `username` as `admin` and password `admin123`. 
+
+__IMPORTANT__: It is highly recommended that you change the `admin` password on first login.
+
+## Nexus Service 
+
+The nexus service is available in namespace `nexus3`:
+
+```shell
+kubectl get service -n nexus3 nexus3-nexus-repository-manager
+```
+
+## Using the Service
+
+### Outside the cluster
+
+__NOTE__: The service does not configured with Public IP, please use Kubernetes Ingress to expose the service to outside world.
+
+Create an ingress:
+
+```shell
```

Comment:
instead of a shell command just change it to normal ingress definition Else everything LGTM

Reply:
what you mean by normal ingress def? plain vanilla YAML ?

Reply:
like you can create an ingress using following code 
`apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: nexus3
  namespace: nexus3
spec:
  rules:
  - http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: nexus3-nexus-repository-manager
            port:
              number: 8081`

Reply:
No worries! thought `cat <<EOF >>` is a much easier way. But updated as you guys might have some standard :)

Reply:
actually not everyone is on linux based os many are on windows, so it's good to be universal